app/Http/Resources/V1/Admin/LmsSettingCollection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Resources\V1\Admin;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class LmsSettingCollection extends ResourceCollection
+{
+    /**
+     * Transform the resource collection into an array.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'data' => $this->collection
+        ];
+    }
+}
```

Comment:
Do we need collection Resource?

Reply:
Yes, I'm using it for including paginated links info in response.